apps/frontend/src/views/Login.vue

```diff
@@ -1,5 +1,8 @@
 <template>
```

Comment:
Additional comments for minor issues I found in this file.
Line 55 - import {SnackbarModule} from '../store/snackbar'; - can we use the @ here
Line 83 - Message not grammatically correct, should use "a problem occurred" not "an problem occurred"